src/client/modules/my/resultsFetcher/resultsFetcher.ts

```diff
@@ -0,0 +1,101 @@
+interface GameResultsData {
+    id: number;
+    game_label: string;
+    player_num: number;
+    player_name: string;
+    victory_points: number;
+}
+
+interface PlayerStatsAllGames {
+    player_name: string;
+    num_games: number;
+    percent_played: string;
```

Comment:
I think it'd be good to defer percents into a string into the UI component. If we have a use-case where we want to use the percent to calculate something else, we couldn't use this value. We could also simplify this interface by making use-cases calculate the percents like percent_first = first_place / total_games.